realtek: switch to 6.12 as default and drop 6.6

jonasjelonek opened the pull request:
thanks to the great work done by @plappermaul and others, 6.12 as testing is already running stable. No issues or other for the last days, so let's switch to 6.12 as default :)

jonasjelonek committed:
realtek: switch to 6.12 as default

Use Linux 6.12 as default for all subtargets.

Signed-off-by: Jonas Jelonek <jelonek.jonas@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19139
Signed-off-by: Robert Marko <robimarko@gmail.com>

plappermaul reviewed (approved):
Ok for me. I don't need that any more.

GoetzGoerisch reviewed (approved):
Thank you for doing this!

andrewjlamarche commented:
6.12 working well for me on (not-yet-officially-supported) S1100WP-8XGT-SE.

jwh reviewed (approved):
Been running with 6.12 for some time, everything looks good

robimarko commented:
Please rebase and I will merge this

jonasjelonek committed:
realtek: remove patches, files and config for 6.6

Remove all files etc. for 6.6 because 6.12 is default now.

Signed-off-by: Jonas Jelonek <jelonek.jonas@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19139
Signed-off-by: Robert Marko <robimarko@gmail.com>
